src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -528,6 +533,87 @@ private JsValue AppendTimeSeries(JsValue document, JsValue name, JsValue[] args)
                 return Undefined.Instance;
             }
 
+            private JsValue IncrementTimeSeries(JsValue document, JsValue name, JsValue[] args)
+            {
+                AssertValidDatabaseContext("timeseries(doc, name).increment");
+
+                const string signature2Args = "timeseries(doc, name).increment(timestamp, values)";
+
+                if(args.Length != 2)
```

Comment:
To match the client we need to have `timeseries(doc, name).increment(value/s)` overload.

Reply:
We support `value/s`, but the timestamp is needed (to my understanding) as in append - I assume it is related to the `DateTime.Now` implementation which is different in JS

Reply:
We also want to support overloading of this and specify a time as well?

Reply:
@ayende do you mean support for append as well?

Reply:
I mean, `timeseries(doc, name).increment(value/s)` and `timeseries(doc, name).increment(date, value/s) `

Reply:
done